spectralevents.py

```diff
@@ -195,13 +195,15 @@ def _energyvec(f, s, Fs, width=7.):
     sf = f / width
     st = 1 / (2 * np.pi * sf)
 
-    t = np.arange(-3.5 * st, 3.5 * st, dt)
-    m = _morlet(f, t, width)
+    t_single_side = np.arange(0., 3.5 * st, dt)
+    # mirror about 0; always an odd # of elements
+    times = np.r_[-t_single_side[-1:0:-1], t_single_side]
```

Comment:
Note that previously, the center of the Morlet wavelet (i.e., at time 0) wasn't necessarily aligned with the signal sample point during convolution.

Reply:
Got it, this edit makes sense then for forcing wavelet alignment to timepoint 0 (alongside the parallel change in the Matlab script).